### PR TITLE
[MMCA-4113] Adding missing fieldset in other journey

### PR DESCRIPTION
**Pull request by wg-hmrc**

### CODE REVIEW

File: app/uk/gov/hmrc/customs/emailfrontend/views/components/inputRadio.scala.html [hunk 1]
```diff
@@ -28,13 +28,7 @@
...
     classes = s"${if(inline){"govuk-radios--inline"}}",
     name = name,
-    fieldset = Some(Fieldset(
-        legend = Some(Legend(
-            content = Text(messages(legend)),
-            isPageHeading = legendAsHeading,
-            classes = classes
-        ))
-    )),
+    fieldset = None,
```
Comment (tyagikumarvineet) [inline app/uk/gov/hmrc/customs/emailfrontend/views/components/inputRadio.scala.html:31]: Could you please write a test for this?

File: app/uk/gov/hmrc/customs/emailfrontend/views/check_your_email.scala.html [hunk 2]
```diff
@@ -39,29 +40,34 @@
 
     @helper.form(CheckYourEmailController.submit, Symbol("id") -> "confirmEmailForm", Symbol("novalidate") -> "novalidate") {
         @CSRF.formField
-        @h1_xl(messages("customs.emailfrontend.confirm-your-email.header", email))
 
-        @p(Html(messages("customs.emailfrontend.confirm-your-email.hint")))
+        @fieldset(
```
Comment (tyagikumarvineet) [inline app/uk/gov/hmrc/customs/emailfrontend/views/check_your_email.scala.html:44]: Could you please write a test to check the existence of this component as we are adding a new component in the page? Test (to check this component) in the controller will server the purpose.

Comment (wg-hmrc) [reply app/uk/gov/hmrc/customs/emailfrontend/views/check_your_email.scala.html:44]: Can this be added in the next story ?

Comment (tyagikumarvineet) [reply app/uk/gov/hmrc/customs/emailfrontend/views/check_your_email.scala.html:44]: Thats fine Waigwa. Could we please make sure that it has been added in the tech debt?

File: app/uk/gov/hmrc/customs/emailfrontend/views/components/inputRadio.scala.html (hunk 1 above)
Comment (tyagikumarvineet) [reply app/uk/gov/hmrc/customs/emailfrontend/views/components/inputRadio.scala.html:31]: It will done as part of tech debt

**Review by tyagikumarvineet — APPROVED** [review]
Suggested tests to be added as part of Tech debt

**Pull request merged**